openlabcmd/openlabcmd/cli.py

```diff
@@ -329,10 +332,14 @@ def _initConfig(self):
                                      "%s" % locations)
 
     def _main(self):
-        self.parser = self.create_parser()
-        self.args = self.parser.parse_args()
-        self._initConfig()
-        self.run()
+        try:
+            self.parser = self.create_parser()
+            self.args = self.parser.parse_args()
+            self._initConfig()
+            return self.run()
+        except exceptions.ClientError as e:
```

Comment:
We should catch all kinds of openlabcmd exception, not only ClientError , but also ValidationError. I think we can add a base Exception there and catch it.

Reply:
done done done